presto-proxy/pom.xml

```diff
@@ -191,4 +191,20 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.basepom.maven</groupId>
+                <artifactId>duplicate-finder-maven-plugin</artifactId>
+                <configuration>
+                    <ignoredResourcePatterns>
+                        <ignoredResourcePattern>.gitkeep</ignoredResourcePattern>
```

Comment:
Added exclusion because .gitkeep files were causing issues 
Added exclusion because H3 classes were causing duplicate classes

```
[WARNING] Found duplicate (but equal) resources in [com.sun.istack:istack-commons-runtime:3.0.7, com.uber:h3:4.0.0]:
[WARNING]   .gitkeep
[WARNING] Found duplicate classes/resources in test classpath.
Also presto-main is only used in test
```
The other exclusions are derived from airlift